tests/queries/0_stateless/03350_alter_table_fetch_partition_thread_pool.sql

```diff
@@ -0,0 +1,19 @@
+-- Tags: no-parallel, no-replicated-database
+-- Tag: no-parallel - to avoid polluting FETCH PARTITION thread pool with other fetches
+-- Tag: no-database-replicated - replica_path is different
+
+drop table if exists data1;
+drop table if exists data2;
+
+create table data1 (key Int) engine=ReplicatedMergeTree('/tables/{database}/{table}', 'r1') order by ();
+create table data2 (key Int) engine=ReplicatedMergeTree('/tables/{database}/{table}', 'r1') order by ();
```

Comment:
Don't we need a different replica name?
```suggestion
create table data1 (key Int) engine=ReplicatedMergeTree('/tables/{database}/{table}', 'r1') order by ();
create table data2 (key Int) engine=ReplicatedMergeTree('/tables/{database}/{table}', 'r2') order by ();
```